gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/GremlinClientTests.cs

```diff
@@ -68,6 +69,20 @@ public async Task ShouldHandleBigResponse()
             }
         }
 
+        [Fact]
+        public async Task ShouldHandle_JToken_Request()
```

Comment:
Another nit, please rename the test to be consistent with the other ones, e.g., no underscores (except when they just describe the tested traversal which isn't the case here) and the test name could a bit more specific. How about something like `ShouldReturnResultWithoutDeserializingItForJTokenType`?

Reply:
Done.